app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
```

Comment:
Looks like they're a holdover from when we were doing PDF manipulation here rather than in `app/services/fast_track/hypertension_pdf_generator.rb`. They're still present in `master`, so feel free to submit a PR to remove them.

Reply:
Done in PR #8972.